src/notification/notification.ui

```diff
@@ -37,7 +37,6 @@
                     <property name="orientation">vertical</property>
                     <child>
                       <object class="GtkOverlay" id="overlay">
-                        <property name="width-request">500</property>
```

Comment:
Removing this property was sufficient for the notification to always expand to the width of the box.